en/mkdocs.yml

```diff
@@ -1518,6 +1518,7 @@ nav:
                 - CI/CD for APIs - Overview: install-and-setup/setup/api-controller/ci-cd-with-wso2-api-management.md
                 - Build a CI/CD Pipeline for APIs Using the CLI: install-and-setup/setup/api-controller/cicd-using-cli.md
                 - Build a CI/CD Pipeline for APIs Using Jenkins: install-and-setup/setup/api-controller/building-jenkins-ci-cd-pipeline-for-dev-first-approach.md
```

Comment:
```suggestion
```
We can remove this page, since this is not applicable for the releases from apictl 4.0.x onwards.